packages/credential-showcase-openapi/openapi/openapi.yaml

```diff
@@ -2608,53 +2749,177 @@ components:
               example: 123e4567-e89b-12d3-a456-426614174123
       description: Anonymous Credentials specific revocation information
 
-    # Showcase:
-    #   type: object
-    #   required:
-    #     - name
-    #     - description
-    #   properties:
-    #     id:
-    #       type: string
-    #       description: Unique identifier for the showcase
-    #       example: 123e4567-e89b-12d3-a456-426614174456
-    #     name:
-    #       type: string
-    #       description: Name of the showcase
-    #       example: Credential Showcase BCGov
-    #     description:
-    #       type: string
-    #       description: Description of the showcase
-    #       example: Collection of credential usage scenarios
-    #     status:
-    #       type: string
-    #       enum: [pending, active, archived]
-    #       description: Current status of the showcase
-    #       example: pending
-    #     hidden:
-    #       type: boolean
-    #       description: Whether the showcase is hidden from public view
-    #       default: false
-    #     scenarios:
-    #       type: array
-    #       description: List of scenarios in this showcase
-    #       items:
-    #         $ref: '#/components/schemas/Scenario'
-    #     credentials:
-    #       type: array
-    #       description: List of credential definitions used in this showcase
-    #       items:
-    #         $ref: '#/components/schemas/CredentialDefinitionsResponse'
-    #     personas:
-    #       type: object
-    #       description: List of personas involved in this showcase
-    #       items:
-    #         $ref: '#/components/schemas/PersonasResponse'
-    #     workflows:
-    #       type: array
-    #       description: List of workflows used in this showcase
-    #       items:
-    #         $ref: '#/components/schemas/Workflow'
+    CredentialSchema:
+      type: object
+      required:
+        - id
+        - name
+        - version
+        - attrNames
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the credential schema
+          example: did:sov:XUeUZauFLeBNofY3NhaZCB
+        name:
+          type: string
+          description: Name of the credential schema
+          example: Diploma Schema
+        issuerId:
```

Comment:
Should this go? Or can we lookup the issuer in the controller and link it?

Reply:
Where in the path /credential-definitions/{definitionId}: according to you is the issuerId then? Since we have an abstract concept of schema's and definitions it can also be used by RPs and thus is not scoped in the path to them. But it can be removed, because that is why we have the identifier in here. That will be automatically the issuer identifier in the context of a schema

Reply:
Was deleted